MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -324,6 +325,33 @@ public RasterizerState RasterizerState
             }
         }
 
+        /// <summary>
+        /// Blend factor for alpha blending.
+        /// </summary>
+        public Color BlendFactor
+        {
+            get { return _blendFactor; }
+            set
+            {
+                if(_blendFactor == value)
+                    return;
+
+                _blendFactor = value;
+
+                var newBlendState = _blendState.Clone();
+
+                newBlendState.BlendFactor = value;
+
+                newBlendState.BindToGraphicsDevice(this);
+
+                _blendState = newBlendState;
+
+                _actualBlendState = newBlendState;
+
+                _blendStateDirty = true;
+            }

```

Comment:
This PR has the right idea, but implements it wrong.

You can see here how the blend factor is a separate DirectX API from the blend state:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/States/BlendState.DirectX.cs#L46

Same with OpenGL:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/States/BlendState.OpenGL.cs#L41

We need to move the blend factor setting completely into the `GraphicsDevice` and apply it during `PlatformApplyState`.

Then the setter for `GraphicsDevice.BlendState`:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/GraphicsDevice.cs#L330

... should simply do:

```
this.BlendFactor = _blendState.BlendFactor;
```
I suspect if you test this all via a unit test you'll see that setting `GraphicsDevice.BlendState` changes the value of `GraphicsDevice.BlendFactor`.